k8s/helm/templates/deployment.yaml

```diff
@@ -121,3 +121,5 @@ spec:
               value: "{{- .Values.newRelic.licenseKey }}"
             - name: NEW_RELIC_APP_NAME
               value: "{{- .Values.newRelic.appName }}"
+            - name: CONFIRMATION_EMAIL_REQUEST_LIMIT
+              value: "10"
```

Comment:
You should make this configurable using `Values` as there is potential we want to increase this in prod etc.